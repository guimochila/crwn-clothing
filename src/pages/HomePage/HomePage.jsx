import React from 'react';
import { HomePageContainer } from './HomePage.styles';
import Directory from '../../components/Directory';

function HomePage() {
  return (
    <HomePageContainer>
      <Directory />
    </HomePageContainer>
  );
}

export default HomePage;
