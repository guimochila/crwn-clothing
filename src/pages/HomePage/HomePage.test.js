import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './HomePage';

it('should render HomePage component', () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
