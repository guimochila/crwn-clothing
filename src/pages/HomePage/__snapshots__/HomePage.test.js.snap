// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage component 1`] = `ShallowWrapper {}`;
